etl/assets.py

```diff
@@ -0,0 +1,71 @@
+import json
+import os
+from pathlib import Path
+
+from dagster import asset, define_asset_job
+
+from etl.resources import (
+    OpenAiEmbeddingModelResource,
```

Comment:
Great naming here.

Reply:
Thank you, but I have renamed these as `OpenAiEmbeddingModelPipeline` and `OpenAiGenerativeModelPipeline`.